token-lending/js/src/instructions/depositReserveLiquidity.ts

```diff
@@ -39,12 +39,15 @@ export const depositReserveLiquidityInstruction = (
         { pubkey: reserveLiquiditySupply, isSigner: false, isWritable: true },
         { pubkey: reserveCollateralMint, isSigner: false, isWritable: true },
         { pubkey: lendingMarket, isSigner: false, isWritable: false },
-        { pubkey: lendingMarketAuthority, isSigner: false, isWritable: false },
         { pubkey: transferAuthority, isSigner: true, isWritable: false },
         { pubkey: SYSVAR_CLOCK_PUBKEY, isSigner: false, isWritable: false },
         { pubkey: TOKEN_PROGRAM_ID, isSigner: false, isWritable: false },
     ];
 
+    if (lendingMarketAuthority) {
+        keys.push({pubkey: lendingMarketAuthority, isSigner: false, isWritable: false})
+    }
+
```

Comment:
Not sure if this is actually needed but wanted to maintain backwards compatibility just in case. Happy to remove if deemed unnecessary!

Reply:
We're better off just removing the parameter totally -- flipping the parameters will cause strange behavior, so let's instead break people totally when they upgrade.